### PR TITLE
testing/rdedup: more arches

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: testing/rdedup/APKBUILD [hunk 1]
```diff
@@ -5,7 +5,7 @@ pkgver="3.1.1"
 pkgrel=1
 pkgdesc="Data deduplication with compression and public key encryption"
 url="https://github.com/dpc/rdedup"
-arch="x86_64 armhf armv7" # limited by cargo
+arch="x86_64 armv7 armhf aarch64 !x86 ppc64le" # limited by cargo (!x86 due linking error in cc package)
```
Comment (Cogitri) [inline testing/rdedup/APKBUILD]: I guess the -fno-stack-protector thingie could fix that too

**Comment by Cogitri** [comment]
CI isn't happy because `license` is missing.

On an unrelated note, are you working on LLVM9 already? Otherwise I'll do the bump tomorrow.

File: testing/rdedup/APKBUILD (hunk 1 above)
Comment (J0WI) [reply testing/rdedup/APKBUILD]: Done

**Comment by Cogitri** [comment]
6d13e33810edfd9b5d12b0b8f7bc1bcf73f43f5e

**Pull request closed without merging**